.github/workflows/release.yaml

```
@@ -34,32 +36,52 @@ jobs:
 
           exit 1;
 
+  update-refs:
+     name: Update References to the latest tag
+     needs: check-images
+     runs-on: ubuntu-latest
+     outputs:
+      tag: ${{ needs.check-images.outputs.tag }}
+     steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.ref }}
+
+      - name: Update references
+        run: |
+          sed -r -i 's/(ref=[a-z0-9]*)/ref='"${{ needs.check-images.outputs.tag }}"'/g' `grep 'ref=' -rl *`
+          sed -r -i '/raw.githubusercontent.com/ s/(deployments-k8s\/[a-z0-9]*)/deployments-k8s\/'"${{ needs.check-images.outputs.tag }}"'/g' `grep 'raw.githubusercontent.com' -rl *`
+          git config --global user.email "nsmbot@networkservicmesh.io"
+          git config --global user.name "NSMBot"
+          git add -- .
+          git commit -s -m "Update references to the latest tag"
+          ref=${{ github.event.ref }}
+          git push origin ${ref#refs/heads/}
+
   create-release:
-    needs: check-images
     name: Create release
+    needs: update-refs
     runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ needs.update-refs.outputs.tag }}
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
         with:
           ref: ${{github.event.ref}}
-      - name: Get tag
-        run: |
-          branch=${{github.event.ref}}
-          echo '::set-output name=tag::'${branch#refs/heads/release/}
-        id: get-tag-step
-      - name: Push tag ${{ steps.get-tag-step.outputs.tag }}
+      - name: Push tag ${{ needs.update-refs.outputs.tag }}
         run: |
           git status
-          git tag ${{ steps.get-tag-step.outputs.tag }}
-          git push origin ${{ steps.get-tag-step.outputs.tag }} -f
-      - name: Create release ${{ steps.get-tag-step.outputs.tag }}
+          git tag ${{ needs.update-refs.outputs.tag }}
```

`needs` looks not correct.

```suggestion
          git tag ${{ jobs.update-refs.outputs.tag }}
```

---

Apply this for other places.